README.md

```diff
@@ -48,49 +48,8 @@ To find out how to build from source and install yourself, please see below.
 * with Snap: see [here](docs/INSTALL.md#snap).
 * with Flatpak: see [here](docs/INSTALL.md#flatpak).
 * with AppImage: see [here](docs/INSTALL.md#appimage).
-* for Mobian: see [here](docs/INSTALL.md#mobian-or-debian-arm64-systems).
-
-### Manually
-
-Requires Rust>=1.71, Cargo and node/npm.
-
-If running Ubuntu, these steps should get you started.
-
-*Note: Make sure to install the latest **node lts** version from [https://nodejs.org/](https://nodejs.org/).*
-
-
-```shell
-make build-dependencies
-```
-
-Now we are good to go. To start, simply use the following:
-
-```shell
-make run
-```
-
-When setting up for the first time and maybe occasionally later you need to update the browser list with your installed browsers.
-
-- change into the `axolotl-web` subfolder
-- run the following command: `npx browserslist@latest --update-db`
-
-## Run flags
-
-* `-axolotlWebDir` Specify the directory to use for axolotl-web. Defaults to "./axolotl-web/dist".
-* `-e` for either
-    `lorca`-> native chromium (has to be installed),
-    `ut` -> runs in the ut enviroment,
-    `me` -> qmlscene,
-    `server` -> just run the webserver. Defaults to run with `electron`.
-* `-eDebug` show developer console in electron mode
-* `-version` Print version info
-* `-host` Set the host to run the webserver from. Defaults to localhost.
-* `-port` Set the port to run the webserver from. Defaults to 9080.
-
-## Environment variables
```

Comment:
it still has run flags, we need to update them and not only delete it.

Reply:
https://github.com/nanu-c/axolotl/blob/c248f457ec1372d9cfa1c8f5d91e095d969f4f2f/src/main.rs#L9-L15

Reply:
Done.